### PR TITLE
Three new map extras

### PR DESCRIPTION
**Pull request by RarkGrames**
#### Summary
```SUMMARY: Content "Added 3 new map extras"```

#### Purpose of change
Adding three new map extras.

#### Describe the solution
prison bus 03
Prison bus verson 1

prison bus 02
Prison bus version 2

prison bus
Prison bus version 3

wasp
Wasp nest

dermatik
Dermatik nest

#### Testing
Spawned all of them to see if everything is correct.

### CODE REVIEW

File: data/json/mapgen/map_extras/prison_bus.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_prison_bus",
+    "weight": 100,
```
Comment (ZhilkinSerg) [inline data/json/mapgen/map_extras/prison_bus.json]: update mapgen cannot have weight node
```suggestion
```

File: data/json/mapgen/map_extras/prison_bus.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        { "monster": "GROUP_ZOMBIE_PRISON", "x": [ 0, 23 ], "y": [ 0, 23 ], "repeat": [ 1, 5 ], "density": 0.1 },
+        { "monster": "GROUP_ZOMBIE_COP", "x": [ 0, 23 ], "y": [ 0, 23 ], "repeat": [ 0, 1 ], "density": 0.1 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_prison_bus",
+    "weight": 100,
```
Comment (ZhilkinSerg) [inline data/json/mapgen/map_extras/prison_bus.json]:
```suggestion
```

File: data/json/mapgen/map_extras/prison_bus.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        { "monster": "GROUP_ZOMBIE_PRISON", "x": [ 0, 23 ], "y": [ 0, 23 ], "repeat": [ 1, 5 ], "density": 0.1 },
+        { "monster": "GROUP_ZOMBIE_COP", "x": [ 0, 23 ], "y": [ 0, 23 ], "repeat": [ 0, 1 ], "density": 0.1 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_prison_bus",
+    "weight": 200,
```
Comment (ZhilkinSerg) [inline data/json/mapgen/map_extras/prison_bus.json]:
```suggestion
```

**Comment by RarkGrames** [comment]
Are they going to spawn with the same chance?

**Comment by ZhilkinSerg** [comment]
> 
> 
> Are they going to spawn with the same chance?

I am pretty sure we only support a single update_mapgen function with a given id.

**Comment by RarkGrames** [comment]
I encountered all three version of it while testing, so it seemed to me that it was working.

**Comment by ZhilkinSerg** [comment]
> 
> 
> I encountered all three version of it while testing, so it seemed to me that it was working.

Then each entry must have the same default weight.

**Comment by RarkGrames** [comment]
Ok, I don't think it's a problem for them to be not weighted, they are not that different.

**Comment by ZhilkinSerg** [comment]
You can change them unique ids and control weight via map extras, but we should probably just add weights for nested/update_mapgens in the future.

**Comment by RarkGrames** [comment]
It can wait until the update_mapget gets weights. After all that will be more similar to how overmap mapgen works.

**Pull request merged**